db/data/20200612054906_copy_teleconsultation_phone_numbers_to_json.rb

```diff
@@ -1,5 +1,6 @@
 class CopyTeleconsultationPhoneNumbersToJson < ActiveRecord::Migration[5.2]
   def up
+    # This data migration did not work correctly, the `teleconsultation_phone_numbers=` method didn't handle string keys
```

Comment:
Neat. Let's add the filename of the working migration to this comment too for good measure.